pants_plugins/BUILD

```diff
@@ -1,2 +1 @@
-pants_requirement(name="pants", dist="pantsbuild.pants")
-pants_requirement(name="pants.testutil", dist="pantsbuild.pants.testutil")
+pants_requirements(name="pants")
```

Comment:
The `pants_requirements` macro now automatically handles configuring both `{package}` and `{package}.testutil`: https://www.pantsbuild.org/docs/plugins-overview#building-in-repo-plugins-with-pants